src/main/java/cheese/CheeseException.java

```diff
@@ -0,0 +1,14 @@
+package cheese;
+
+/**
+ * Custom exception
```

Comment:
JavaDoc header comments should start in the form of something like 'Customizes...' or 'Represents...'